src/main/java/org/apache/jackrabbit/filevault/maven/packaging/ValidatePackageMojo.java

```diff
@@ -64,14 +67,30 @@
     /** If set to {@code true} will not validate any sub packages. This settings overwrites the parameter {@code enforceRecursiveSubpackageValidation}. */
     @Parameter(required = true, defaultValue = "false")
     private boolean skipSubPackageValidation;
-    
+
+    /**
+     * If given, the classifier will be used to get the package to be verified from the attached artifacts from the project with the same
+     * classifier.
+     */
+    @Parameter
+    private String classifier;
```

Comment:
why do you want to have two artifacts for a single module in the first place?

Reply:
Also if you want to validate anything apart from the project primary artifact you can already use parameter `packageFile` for that.

Reply:
In our use case we want to have two artifacts for the same content-package as for a cloud based deployment the content should be installed on different path as for the other OnPrem deployments.

Reply:
Why can't you work with different packageFiles then? Maybe you can give an example project where you want to leverage this extension. I am currently not really understanding what you want to do....

Reply:
@kwin , we're considering creating an alternate build of Core Components, based on the same sources but located in `/apps`, released as a separate package with a `-cloud` classifier (see https://github.com/adobe/aem-core-wcm-components/pull/1002/files).

We looked at using the `finalName` (see https://github.com/adobe/aem-core-wcm-components/pull/1003/files) and that works for the filevault plugin but we will need to duplicate configuration for install and deploy/release plugins as well. It would be easier for us if the filevault plugin would support classifier.

Reply:
Can you share the code? Who attached the 2nd artifact then? Are you using https://www.mojohaus.org/build-helper-maven-plugin/attach-artifact-mojo.html? Would it make sense to just find the right artifact from the attached ones instead of creating a file name out of the classifier?

Reply:
BTW, I tried to attach the `-cloud` artifact (https://github.com/adobe/aem-core-wcm-components/pull/1003/commits/7af5f6f23a9d59ab4aaaeaaf305bdeff3ce920da) and it gets picked up for install but not for deploy.

Reply:
Yes, the reason is that each execution of the `package` goal will overwrite the primary artifact. If you want to support multiple artifacts in the same module you need to change these lines as well: https://github.com/apache/jackrabbit-filevault-package-maven-plugin/blob/697caf92e4c6a8a6be9fa0447b22766d138fa2e0/src/main/java/org/apache/jackrabbit/filevault/maven/packaging/VaultMojo.java#L483. For attaching secondary artifacts please use https://maven.apache.org/ref/3.2.3/apidocs/org/apache/maven/project/MavenProjectHelper.html#attachArtifact(org.apache.maven.project.MavenProject,%20java.io.File,%20java.lang.String).

Reply:
Right, I think we can work with the attach-artifacts plugin to get the expected output. Will close this PR

Reply:
That was not what I was intending to say. The PR is useful but not complete. IMHO you need to:
1. Make sure you don't overwrite the file with secondary artifacts
2. Attach secondary artifacts correctly to the Maven Project
3. For `validate-package` just validate all attached packages

Reply:
@kwin Thank you for your feedback. I create a new PR for this feature at https://github.com/apache/jackrabbit-filevault-package-maven-plugin/pull/40 with the requested changes.